pkg/login/social/social.go

```diff
@@ -123,6 +123,7 @@ func ProvideService(cfg *setting.Cfg,
 		conn, err := ss.createOAuthConnector(name, settingsKVs, cfg, features, cache)
 		if err != nil {
 			ss.log.Error("Failed to create OAuth provider", "error", err, "provider", name)
+			continue
```

Comment:
Nit. We should totally do this, but I realize there might be other "problems" introduced with this perhaps? So should we perhaps do this in a follow up/add config/feature toggle that can be turned off :shrug: It's your call

Reply:
Alright, so just to be safe I remove this change and only add the parsing (which should fix the issue)